app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,40 @@
+class ReviewsController < SecuredController
+  skip_before_action :authorize_request, only: %i[index show]
+
+  # GET /reviews
+  def index
+    user_sub = request.headers['body']
```

Comment:
I'm curious about why we need to place user_sub in the request headers and not directly in the request parameters (aka as part of the request URL)? Normally we use the headers to store stuff like cookies, tokens, and other meta info about the request and not any request data itself.

For POST and PUT/PATCH requests we normally use the request body to hide the data.

Reply:
Hello @rvvergara 

In order for me to get reviews, I need to pass in the authenticated user's unique id (given by auth0), which is user_sub in my case. Moreover, I cannot send data as a request body with the GET request. therefore I thought it was appropriate to send it in headers instead of as request parameters.

Reply:
Understood @tanzila-abedin. However, I think for future projects we can simplify how we do this. In fact, the usage for JWT could have been simpler. I refer you to this [article](https://medium.com/@noordean/authentication-with-jwt-in-rails-api-71102387c0b0) for future reference.

You can also check out some of my own API rails projects on how I used requests to capture data.